content/docs/command-reference/metrics/show.md

```diff
@@ -98,10 +58,60 @@ compares them with a previous version.
 
 ## Examples
```

Comment:
In the examples, wouldn't it be better to include an example with metrics file in format of CSV  or TSV's rather than always using JSON files

Reply:
Those formats are no longer supported by `dvc metrics`.